src/lib/HWTIMER/ESP32_hwTimer.cpp

```diff
@@ -59,8 +59,11 @@ void ICACHE_RAM_ATTR hwTimer::resume()
         isTick = false;
         // When using EDGE triggered timer, enabling the timer causes an edge so the interrupt
         // is fired immediately, so this emulates the STM32 behaviour
-        timerAlarmWrite(timer, HWtimerInterval >> 1, true);
+        // Unlike the 8266 timer, the ESP32 timer can be started without delay.
+        // It does not interrupt the currnelty running IsrCallback(), but triggers immediatly once it has completed.
```

Comment:
(nitpick - feel free to ignore) `currnelty` should be `currently` and `immediatly` -> `immediately` also needs fixing